modules/nf-core/glimpse/ligate/main.nf

```diff
@@ -8,7 +8,7 @@ process GLIMPSE_LIGATE {
         'quay.io/biocontainers/glimpse-bio:1.1.1--hce55b13_1' }"
 
     input:
-    tuple val(meta), path(input)
+    tuple val(meta), path(input_list)
```

Comment:
It would be better to split the files and value inputs in two different channels. (so one channel called `inputs` for example that contains all files and another called `regions` for example). That way all files get staged correctly while the values remains values.

Reply:
I don't quite understand.
In this module I do not have any regions informations.
I just have multiple files that I need to merge in one.
Wouldn't it concern more the phasing modules ?

Reply:
Oh yeah my bad it was `phase` that needed the regions, this is fine then!